modules/administration-guide/pages/calculating-che-resource-requirements.adoc

```diff
@@ -73,7 +89,7 @@ Learn how to calculate resources, such as memory and CPU, required to run {prod}
 |`postgresql`
 |128 MiB
 |64 MiB
-|50 m
+|1000 m
```

Comment:
this and below figures have been fixed as part of https://github.com/eclipse-che/che-docs/pull/2578/files, no?

Reply:
:) Yes, and I started working on fixing these here before you opened https://github.com/eclipse-che/che-docs/pull/2579, and I used this branch when reviewing your PR. This PR just needs a rebasing now that you've prioritized merging https://github.com/eclipse-che/che-docs/pull/2579/files.